infra/keyVault.bicep

```diff
@@ -0,0 +1,47 @@
+@description('Specifies the name of the key vault.')
+param keyVaultName string = 'tempV22'
```

Comment:
default kv name doesn't follow conventions

Reply:
my bad! it was supposed to be overwritten by the param passed from the main.bicep. I'll update this too